### ISSUE TITLE
build_test_containers is broken because of Poetry

### ISSUE THREAD

**Issue by dgarros** [networktocode/yangify#27]
I tried to build the test container with `make build_test_containers` but I'm getting an error line 7 
```
RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python && \
    /root/.poetry/bin/poetry config settings.virtualenvs.create false && pip install -U "pip<19"
```

The error message report an issue with `settings.virtualenvs.create ` 
```
[ValueError]
Setting settings.virtualenvs.create does not exist
```

Poetry 1.0.0 got released 3 weeks ago so my guess is that option is not supported anymore in 1.0.0

**Comment by FragmentedPacket** [comment]
This appears to be fixed in #25, but we need to review it. Once it's merged, this issue should be resolved.

> The settings. prefix is no longer necessary
> 
> The settings are now stored differently and no longer needs to be prefixed by settings.. If you have already configured settings you will need to configure them again.

From this [link](https://python-poetry.org/blog/announcing-poetry-1-0-0.html).

**Comment by FragmentedPacket** [comment]
@dgarros This should be resolved and appears to be working. Do you mind testing with the latest changes and let me know?
